engine/src/main/java/org/terasology/rendering/dag/nodes/BackdropReflectionNode.java

```diff
@@ -50,7 +50,6 @@
  *
  */
 public class BackdropReflectionNode extends AbstractNode {
-
```

Comment:
If you did it manually this in particular is a matter of personal preferences and I'm not too bothered either ways. The important thing is that it's not some kind of automated functionality you are unaware of. So, no worries on this one.